frontend/packages/console-shared/src/components/getting-started/GettingStartedExpandableGrid.tsx

```diff
@@ -34,27 +38,38 @@ export const GettingStartedExpandableGrid: React.FC<GettingStartedExpandableGrid
     </span>
   );
 
+  const handleClose = () => {
+    setShowState(GettingStartedShowState.HIDE);
+  };
+
   return (
     <ExpandableSection
       onToggle={() => setIsOpen(!isOpen)}
       isExpanded={isOpen}
       displaySize="lg"
       className="ocs-getting-started-expandable-section"
       toggleContent={
-        <div className="ocs-getting-started-expandable-section__toggle-text">
-          <Title headingLevel="h2" size={TitleSizes.lg} data-test="title">
-            {title}{' '}
-            <Popover bodyContent={titleTooltip} triggerAction="hover">
-              <span
-                role="button"
-                aria-label={t('console-shared~More info')}
-                className="ocs-getting-started-expandable-grid__tooltip-icon"
-              >
-                <OutlinedQuestionCircleIcon />
-              </span>
-            </Popover>
-          </Title>
-        </div>
+        <>
```

Comment:
nit: because the entire content is wrapped with a div, the react.fragment isn't necessary

Reply:
Resolved it. PTAL.